src/clarity-angular/popover/signpost/_signposts.clarity.scss

```diff
@@ -24,8 +24,26 @@
                 height: $clr_baselineRem_1;
             }
 
-            &:hover, &.active {
+            &.active {
                 color: $clr-action-blue;
+                clr-icon {
```

Comment:
:-( Targeting `clr-icon` directly is almost never a good idea because its possible that it might target other icons in that component but in this case I guess its needed because the icons are projected by the user. Is that right?

Reply:
I showed usage of a clr-icon in my documentation on the website but we project anything they give us. 
What about targeting the first child of `.signpost` elements?

Reply:
We could probably attach a class to the trigger directive and target that class on `clr-icon` element, maybe?

Reply:
I'll  take a look and see if that will work.